hardhat/hardhat.config.ts

```diff
@@ -15,6 +15,7 @@ const ENV_FILE = process.env.DOTENV_CONFIG_PATH || '../.env'
 dotenv.config({ path: ENV_FILE })
 
 const NETWORK = process.env.NETWORK || "geth";
+const NETWORK_URL = process.env.WEB3_RPC_URL || 'http://localhost:8545';
```

Comment:
Can we give this a different name? From the way the env var `WEB3_RPC_URL` is used all around it points to the websocket endpoint (which for dev is `localhost:8546`), maybe `WEB3_HTTP_URL`?

Reply:
Yeah good point. Updated it to `WEB3_HTTP_URL`.